BUILD.gn

```diff
@@ -493,14 +494,10 @@ if (is_mac) {
     ]
   }
 
-  if (!is_component_build) {
+  if (is_component_ffmpeg) {
     bundle_data("electron_framework_libraries") {
-      sources = []
```

Comment:
Is there a reason for changing this condition ? The original reason was to have a generic `Libraries` creation under the non-component build, as we may not only ship ffmpeg in future.

Reply:
@deepak1556 my bad... it doesn't need to be changed.  I was trying to fix the issue of  `Electron.app/Contents/Frameworks/Electron Framework.framework/Library` missing  which is a symlink to  Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/Current/Libraries.  It turns out I just needed to make the change at https://github.com/electron/electron/pull/14708/files/c91223de64ec8f6a8f3376a80848eae6e1fdbd2a#diff-4eaebbcd1dc0da11e3a3ce488290bfc6L530